app/sidekiq/lighthouse/create_intent_to_file_job.rb

```diff
@@ -21,44 +26,52 @@ class CreateIntentToFileError < StandardError; end
     # exhausted attempts will be logged in intent_to_file_queue_exhaustions table
     sidekiq_options retry: 14, queue: 'low'
     sidekiq_retries_exhausted do |msg, error|
-      form_type, form_start_date, veteran_icn = msg['args']
+      in_progress_form_id, veteran_icn = msg['args']
+      in_progress_form = InProgressForm.find(in_progress_form_id)
       itf_log_monitor = BenefitsClaims::IntentToFile::Monitor.new
+      form_type = in_progress_form.form_id
       itf_type = ITF_FORMS[form_type]
-      user_account = UserAccount.find_by(icn: veteran_icn)
 
-      itf_log_monitor.track_create_itf_exhaustion(itf_type, form_start_date, user_account&.id, error)
+      itf_log_monitor.track_create_itf_exhaustion(itf_type, in_progress_form, error)
 
       # create ITF queue exhaustion entry
-      itfqe = IntentToFileQueueExhaustion.create({
-                                                   form_type:,
-                                                   form_start_date:,
-                                                   veteran_icn:
-                                                 })
+      exhaustion = IntentToFileQueueExhaustion.create({
+                                                        form_type:,
+                                                        form_start_date: in_progress_form.created_at,
+                                                        veteran_icn:
+                                                      })
       ::Rails.logger.info(
-        "IntentToFileQueueExhaustion id: #{itfqe.id} entry created", {
-          intent_to_file_queue_exhaustion: itfqe
+        "IntentToFileQueueExhaustion id: #{exhaustion.id} entry created", {
+          intent_to_file_queue_exhaustion: exhaustion
         }
       )
     end
 
     ##
-    # Create an Intent to File using the Lighthouse ITF endpoint for given form type and ICN
+    # Create an Intent to File using the Lighthouse ITF endpoint for given in progress form, ICN, and PID
     #
-    # On success do nothing for now
-    # Raises CreateIntentToFileError
+    # On success/failure log and increment the respective Datadog counter
     #
-    # @param [String] form_type
-    # @param [ActiveSupport::TimeWithZone] form_start_date
+    # @param [Integer] in_progress_form_id
     # @param [String] veteran's ICN
+    # @param [String] veteran's participant ID
     #
-    def perform(form_type, form_start_date, veteran_icn)
-      init(form_type, veteran_icn)
+    def perform(in_progress_form_id, veteran_icn, participant_id)
+      raise MissingICNError, 'Init failed. No veteran ICN provided' if icn.blank?
+      raise MissingParticipantIDError, 'Init failed. No veteran participant ID provided' if participant_id.blank?
+
+      init(in_progress_form_id, veteran_icn)
+
+      itf_log_monitor ||= BenefitsClaims::IntentToFile::Monitor.new
+      service ||= BenefitsClaims::Service.new(icn)
```

Comment:
at least the monitor since that is used in the triage function
```suggestion
def itf_log_monitor
      @itf_log_monitor ||= BenefitsClaims::IntentToFile::Monitor.new
end
def service
      @service ||= BenefitsClaims::Service.new(icn)
end
```